service/src/main/resources/landingzonedb/changesets/20221019_billingProfileId.yaml

```diff
@@ -0,0 +1,14 @@
+# The schema for a landing zone
+databaseChangeLog:
+- changeSet:
+    id: add billing profile id field.
+    author: myazykov
+    changes:
+    - addColumn:
+        tableName: landingzone
```

Comment:
Wonder if this should be not-null. I know we have existing records in the DB, but maybe we can put a dummy value or something (it should be required going forward).